ocean_lib/models/test/test_erc20_enterprise.py

```diff
@@ -245,25 +246,22 @@ def test_buy_from_fre_and_order(
         amount=consume_fee_amount,
         from_wallet=publisher_wallet,
     )
+    data_provider = DataServiceProvider
+    _, metadata, encrypted_files = create_basics(config, web3, data_provider)
```

Comment:
The `create_basics` method is for testing purposes only, and probably should not be included in READMEs.